src/components/ui/Textarea.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components/macro'
+
+const Textarea = styled.textarea`
+  border: 1px solid ${({ theme }) => theme.secondaryBackground};
+  box-sizing: border-box;
+  border-radius: 12px;
+  padding: 15px 20px;
+  resize: vertical;
+  color: ${({ theme }) => theme.secondaryText};
+  font-family: inherit;
+  font-size: 18px;
+  width: 100%;
+  height: 140px;
+
+  ::placeholder {
```

Comment:
this style does not work. It compiles to the selector `textarea.className ::placeholder` instead of `textarea.className::placeholder` (note the space to denote a child in the first example).

Change this to `&::placeholder` instead

Reply:
thx for pointing out and explaining